### ISSUE TITLE
Tooltip lingers

### ISSUE THREAD

**Issue by jscheid** [NuCivic/react-nvd3#41]
Hi, not sure if this is related to #40 but have a look here: https://jsfiddle.net/julians37/uazsgvya/

Mouse over an item so that the tooltip is shown, then click.

This causes the chart to be unmounted, yet the tooltip is still visible.

**Comment by topicus** [comment]
It seems like the renderEnd method is never triggered in the scatterChart. Probably a nvd3 issue. Still working in this.

**Comment by grantga** [comment]
Any update on this issue--it seems to still be present in the scatter chart in the latest release. Any ideas on a workaround?

**Comment by grantga** [comment]
Looks like this is a problem of renderEnd not getting called for some chart types in more recent nvd3 releases:
https://github.com/novus/nvd3/issues/1720

Downgrading to nvd3@1.8.1 seemed to fix the renderEnd issue.

**Comment by topicus** [comment]
@grantga @jscheid Sorry I've been out of pocket this weeks. I've tried to fix it but as you pointed out that's an issue in the nvd3 library. Render end event it's not consistent across chart models. Sometimes you need to subscribe to the child component (line, pie, etc.) and sometimes to the parent (lineChart, pieChart, etc.) and sometimes neither of them works.

**Comment by jscheid** [comment]
Fair enough, thanks for the update. I'd rather not downgrade but I'll see if I can have a go at fixing the upstream issue one of these days.
